machine/v1beta1/types_vsphereprovider.go

```diff
@@ -51,6 +51,11 @@ type VSphereMachineProviderSpec struct {
 	// This parameter will be ignored if 'LinkedClone' CloneMode is set.
 	// +optional
 	DiskGiB int32 `json:"diskGiB,omitempty"`
+	// TagIDs is an optional set of tags to add to an instance. Specified tagIDs
+	// must use URN-notation instead of display names.
+	// For example: urn:vmomi:InventoryServiceTag:5736bf56-49f5-4667-b38c-b97e09dc9578:GLOBAL
```

Comment:
Where will this format be validated?

Reply:
At present, we are relying on the vCenter API to perform validation during reconciliation but I'll add regex to enforce this in the CRD.

Reply:
Introduced validation in the [machine webhook](https://github.com/openshift/machine-api-operator/pull/1182/commits/2137696e9d2f30f1aa3e7e9f96b5c229196bd59f#diff-30c686f7a9dbd2aa917e5e410401d720a2a4bf4a11163495f5610c8f4b98edaeR121-R1435)